contracts/DepositManager.sol

```diff
@@ -119,7 +127,12 @@ contract DepositManager is DepositCore, IDepositManager {
         vault = _vault;
     }
 
-    function setRollupAddress(address _rollup) public {
+    /**
+     * @notice Sets Rollup contract address. Can only be called once by owner
+     * @dev We assume DepositManager is deployed before Rollup
+     * @param _rollup Rollup contract address
+     */
+    function setRollupAddress(address _rollup) external initializer onlyOwner {
```

Comment:
Ownable emits events and deploys public functions we don't use. What do you think we use the handcraft version? 

```
address private immutable _deployer;

constructor(){
    _deployer = msg.sender;
}

function setRollupAddress(address _rollup) external initializer {
    require(msg.sender==_deployer, "Only deployer");
    ...
}
```